### PR TITLE
Trev8939/camera viewshed

### PR DESCRIPTION
**Pull request by TADraeseke**
Adds 100.2 sample viewshed from a camera in a scene view.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,6 @@
 Icon?
 ehthumbs.db
 Thumbs.db
+
+# Android Studio
+.idea/
```
Comment (shellygill) [inline .gitignore]: We already have a number of IntelliJ files in the Java and Kotlin subfolder .gitignore files. Potentially we can move these items all up to this root .gitignore, but we don't need it to be duplicated.

File: java/viewshed-camera/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.esri.arcgisruntime.sample.viewshedcamera">
+
+    <uses-permission android:name="android.permission.INTERNET" />
+
+    <uses-feature
+        android:glEsVersion="0x00020000"
```
Comment (shellygill) [inline java/viewshed-camera/src/main/AndroidManifest.xml]: Requires OpenGL ES 3 or above.

File: .gitignore (hunk 1 above)
Comment (TADraeseke) [reply .gitignore]: Good point. I'll look into it as a seperate PR.

**Comment by TADraeseke** [comment]
Thanks @shellygill. Both suggestions implemented.

File: version.gradle [hunk 3]
```diff
@@ -11,11 +11,11 @@ ext {
...
+    arcgisVersion = '100.2.0-1890'
     arcgisToolkitVersion = '100.1.0-SNAPSHOT'
     // plugin versions
     gradleVersion = '3.0.0'
-    // java versions
-    targetJavaVersion = 1.8
-    sourceJavaVersion = 1.8
+	// java versions
+	targetJavaVersion = 1.8
+	sourceJavaVersion = 1.8
```
Comment (shellygill) [inline version.gradle]: Formatting here is using tabs instead of spaces.

**Comment by shellygill** [comment]
Thanks @TADraeseke - one minor comment about formatting (I didnt notice this before I'm afraid) then good to merge.

**Pull request merged**